cluster-management/debugging/manual-rollbacks/index.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: docs
+slug: guides
+title: Manual CoreOS Rollbacks
+category: cluster_management
+sub_category: debugging
+weight: 5
+---
+
+# Performing Manual CoreOS Rollbacks
+
+As much as we all love flawless, automatic updates, there may be occasions when
+an update must be rolled back. This is fairly straightforward on CoreOS, once
+you know the right commands.
+
+TL;DR; The following command will set the currently passive partition to be
```

Comment:
If it's an initialism, shouldn't it be capitalized? (you are right about the colon)